src/elements/content-sharing/hooks/useCollaborators.js

```diff
@@ -28,29 +26,21 @@ function useCollaborators(
     api: API,
     itemID: string,
     itemType: ItemType,
-    ownerEmail: ?string,
-    isCurrentUserOwner: boolean,
-    handleSuccess: ?Function,
-    handleError: ?Function,
+    handleSuccess: Function = noop,
+    handleError: Function = noop,
 ): collaboratorsListType | null {
     const [collaboratorsList, setCollaboratorsList] = React.useState<collaboratorsListType | null>(null);
 
     React.useEffect(() => {
         if (collaboratorsList) return;
 
         const handleGetCollaborationsSuccess = (response: Collaborations) => {
-            const updatedCollaboratorsList = convertCollabsResponse(response, ownerEmail, isCurrentUserOwner);
-            setCollaboratorsList(updatedCollaboratorsList);
-            if (handleSuccess) {
-                handleSuccess();
-            }
+            setCollaboratorsList(handleSuccess(response));
```

Comment:
Should we rely on the `handleSuccess` to return the data? What about something like:
```
setCollaboratorsList(response);
handleSuccess(response);
```